configs/machines/levante.yaml

```diff
@@ -275,7 +291,12 @@ mpifc: "$MPIFC"
 mpicc: "$MPICC"
 cxx: "$CXX"
 
-launcher_flags: "-l --distribution=plane=128 --cpu-bind=${cpu_bind}"
+# kh 23.06.22 do not use --distribution=plane=128 for hybrid (MPI + OpenMP) jobs
+# launcher_flags: "-l --distribution=plane=128 --cpu-bind=${cpu_bind}"
+
+# kh 23.06.22 should be set generally in accordance with 
+# the tips from DKRZ-TechTalk "From Mistral to Levante" June 22, 2022
+launcher_flags: "-l ${mt_launcher_flag}" # --distribution=block:cyclic"
```

Comment:
@khimstedt , I moved the `--hint=nomultithread` flag to be added here, so that it only affects `Levante` and not all the other systems we are currently using. Before, you added it to the `launcher_flags_per_component` in the `slurm.yaml` which means that for an heterogeneous parallel `srun` call the flag will be called for each component:
```
srun ... --hint=nomultithreading <component1> :
... --hint=nomultithreading <component2>
```

After my changes --hint=nomultithreading is used only once. Would that be okay for heterogeneous parallelization runs with hetjobs?

Reply:
Hi Kai, I'll work on a solution so that each srun component can get the flag without affecting the other HPCs srun commands.